api/app/signals/settings/development.py

```diff
@@ -9,7 +9,7 @@
 
 SIGNALS_AUTHZ = {
     'JWKS_URL': "http://dex:5556/keys",
-    'ALWAYS_OK': False,
+    'ALWAYS_OK': True,
```

Comment:
Not a blocker, but ...

Do we want to retain the default behavior with regards to the `'ALWAYS_OK'` option? (Probably, but consider it since this looks out of place in this PR.)

Reply:
Ah, yes. Forgot about that. Reverted it!